src/Illuminate/Foundation/Bootstrap/DetectEnvironment.php

```diff
@@ -16,6 +16,11 @@ public function bootstrap(Application $app)
 	{
 		try
 		{
+			if( "mutable" == getenv('VAR_MUT') )
```

Comment:
@Anahkiasen :+1: Didn't know this, but have never seen them in the framework. Makes sense.